### PR TITLE
Changelog for 0.19.0

### PR DESCRIPTION
**Pull request by angelozerr**
Changelog for 0.18.4

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+### Enhancements
+
+- Eliminate need for downloading just-released binary checksums.. See [#650](https://github.com/redhat-developer/vscode-xml/pull/650).
+- Prevent suspicious directory traversal. See [lemminx#1171](https://github.com/eclipse/lemminx/pull/1171).
+- Basic support for parameter entities. See [lemminx#1167](https://github.com/eclipse/lemminx/issues/1167).
+- Support for document link DTD entity SYSTEM. See [lemminx#1165](https://github.com/eclipse/lemminx/issues/1165).
+
+### Bug Fixes
+
+- Bad SYSTEM for DTD DocType and Entity breaks the XML validation. See [lemminx#1169](https://github.com/eclipse/lemminx/issues/1169).
```
Comment (rgrunber) [inline CHANGELOG.md:14]: Can we use the bug on the vscode-xml side for this ? Eg. https://github.com/redhat-developer/vscode-xml/issues/647 . You can always keep the title the same or change to explain better than the original bug.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## [0.18.4](https://github.com/eclipse/lemminx/milestone/26?closed=1) (February 14, 2022)
+
+### Enhancements
+
+- Eliminate need for downloading just-released binary checksums.. See [#650](https://github.com/redhat-developer/vscode-xml/pull/650).
```
Comment (rgrunber) [inline CHANGELOG.md:7]: I would remove the extra "." at the end of "checksums" and place this entry under a "Build" heading.

Comment (angelozerr) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [0.18.4](https://github.com/eclipse/lemminx/milestone/26?closed=1) (February 14, 2022)
```
Comment (rgrunber) [inline CHANGELOG.md:3]: Ah sorry, just noticed this. should be for vscode-xml. Milestone number is correct.

Comment (fbricon) [reply CHANGELOG.md:3]: Given the enhancement, it should probably be 0.19.0

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,22 @@
...
+### Enhancements
+
+ * Basic support for parameter entities. See [lemminx#1167](https://github.com/eclipse/lemminx/issues/1167).
+ * Support for document link DTD entity SYSTEM. See [lemminx#1165](https://github.com/eclipse/lemminx/issues/1165).
+
+### Bug Fixes
+
+ * Bad SYSTEM for DTD DocType and Entity breaks the XML validation. See [#647](https://github.com/redhat-developer/vscode-xml/issues/647).
+ * Prevent suspicious directory traversal. See [lemminx#1171](https://github.com/eclipse/lemminx/pull/1171).
+ * Limit resource downloads to http, https and ftp and prevent insecure redirects. See [lemminx#1174](https://github.com/eclipse/lemminx/pull/1174).
```
Comment (fbricon) [inline CHANGELOG.md:14]: add https://github.com/redhat-developer/vscode-xml/pull/659

Comment (angelozerr) [reply CHANGELOG.md:14]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (rgrunber) [reply CHANGELOG.md]: Fair enough. Let's make it 0.19.0 :)

**Review by fbricon — APPROVED** [review]

**Pull request merged**